src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -62,7 +62,7 @@ public void testGetTablesNameWhenDbHasNotTables() throws SQLException, ClassNotF
         preparationEnvironment();
```

Comment:
Maybe verb instead of noun?

Reply:
At the next pull request i will removing this method